2-mandatory/4-people-i-know.js

```diff
@@ -378,13 +378,21 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
 let thirtyFiveOrOlder = [];
+friends
+  .filter((friend) => friend.age > 34)
+  .forEach((friend) => {
+    thirtyFiveOrOlder.push(friend);
+  });
```

Comment:
you don't need to push array elements after using filter, this code will suffice:
```suggestion
let thirtyFiveOrOlder = friends.filter((friend) => friend.age >= 35);
```